system/include/emscripten/emscripten.h

```diff
@@ -24,6 +24,10 @@ extern "C" {
 #include <SDL/SDL.h> /* for SDL_Delay in async_call */
 #endif
 
+#if __EMSCRIPTEN__
```

Comment:
My memory is fuzzy here, are compilers required to interpret `#if __EMSCRIPTEN__` as false if `__EMSCRIPTEN__` is not defined in the first place? GCC at least does so, but is that same across other compilers? Would `#ifdef __EMSCRIPTEN__` be better? (since we will never have `#define __EMSCRIPTEN__ 0` present)